R/subset.R

```diff
@@ -82,7 +82,7 @@ armadillo.subset <- function(source_project = NULL,
   type <- folder <- . <- NULL
 
   suppressMessages(
-    source_tables <- within(armadillo.list_tables("gecko") %>%
+    source_tables <- within(armadillo.list_tables(source_project) %>%
```

Comment:
Should you test (tests/testthat/test-subset.R) with another name then 'gecko'?

The docs use gecko so it probably is OK